### ISSUE TITLE
[Windows] build progress output broken on Windows since version 6.0

### ISSUE THREAD

**Issue by Emeryao** [angular/angular-cli#11403]
### Bug Report or Feature Request

- [x] bug report
- [ ] feature request

### Area
- [x] devkit
- [ ] schematics

### Versions

* Windows 10 1803 Build 17134.137
* node `10.5.0`
* npm `6.1.0`
* ng version
```
Angular CLI: 6.0.8
Node: 10.5.0
OS: win32 x64
Angular: 6.0.7
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.6.8
@angular-devkit/build-angular     0.6.8
@angular-devkit/build-optimizer   0.6.8
@angular-devkit/core              0.6.8
@angular-devkit/schematics        0.6.8
@angular/cli                      6.0.8
@ngtools/webpack                  6.0.8
@schematics/angular               0.6.8
@schematics/update                0.6.8
rxjs                              6.2.1
typescript                        2.7.2
webpack                           4.8.3
```

### Repro steps
1. create a new project with `ng new`
1. try to build/serve the project on Windows
1. see the build progress output 

> I tried both `powershell` and `cmd` on Windows
> I'm not sure if this is a Windows issue and I also tried on the Windows Insiders as well
> I think the issue Microsoft/vscode#19665 may helps probably

### The log given by the failure
![image](https://user-images.githubusercontent.com/3591899/42072139-971cda84-7b91-11e8-9ab5-64ecafd02efc.png)
![image](https://user-images.githubusercontent.com/3591899/42072144-9dc4c310-7b91-11e8-9f38-d99995f75db5.png)

### Desired functionality
the build progress output should not be broken on Windows

BTW: the progress output works nicely as always in `Bash` on `Ubuntu 16.04`

SORRY for my poor English 😆

**Comment by Emeryao** [comment]
it's `@angular/cli@6.1.3` with `@angular-devkit/build-angular@0.7.3` and the issue is still there 😢

**Comment by filipesilva** [comment]
I'm not sure what's happening, but I am on Windows, using Git Bash, and don't see this behaviour.

**Comment by Emeryao** [comment]
hi @filipesilva 
After doing some experiments, I think this issue is related to the `font setting` of the terminal (powershell, git bash , etc)

when I change the font setting of `Git Bash` like this the build output goes into a mess 
![image](https://user-images.githubusercontent.com/3591899/43953042-618d5560-9cca-11e8-9202-b64d8fb37aaa.png)

but when I change the font setting of `Git Bash` like this the output goes well
![image](https://user-images.githubusercontent.com/3591899/43953200-d007004a-9cca-11e8-83bc-7b1f590eb7a1.png)
> just changed the font size from `8x16` to `8x12`

SORRY for my poor English 😆

**Comment by Emeryao** [comment]
I redo the experiment above on the *old* angular project with `angular@4.4.7` and `@angular/cli@1.7.4`
the output goes very well no matter what the font setting of `Git Bash` is

**Comment by filipesilva** [comment]
Thanks for investigation 👍 

It might be related with changes on Webpack's side as well, we'll have to investigate. Best I can say for now is to turn off progress reporting via `--progress=false` or in the `angular.json` file.

**Comment by Emeryao** [comment]
@filipesilva 
this issue has been fixed with `@angular/cli@6.2.1` and `@angular-devkit/build-angular@0.8.1` in my scenario 🎉
